### ISSUE TITLE
ERROR FREESWITCH  AFTER UPDATING /opt/freeswitch/var/log/freeswitch/freeswitch.log: No such file or directory

### ISSUE THREAD

**Issue by erpsolns** [bigbluebutton/bigbluebutton#12235]
Installation error from (bbb-install.sh)updating from normal working server - on 2.3 rc-1  (PREVIOUSLY no problem)to updating to 2.3 installation error  

**Describe the bug**
3) # Potential problems described below
grep: /opt/freeswitch/var/log/freeswitch/freeswitch.log: No such file or directory
# Error: Unable to connect to the FreeSWITCH Event Socket Layer on port 8021
In The server  room  audio connection fails with "Couldnot make Websocket connection error 1002
**To Reproduce**
r
grep: /opt/freeswitch/var/log/freeswitch/freeswitch.log: No such file or directory
# Error: Unable to connect to the FreeSWITCH Event Socket Layer on port 8021
In The server  room  audio connection fails with "Couldnot make Websocket connection error 1002
**Expected behavior**
 installation without error
Audio Microphone or Audio  Listen only should work normally
Now both of them are not working and the installation is useless as no audio is possible
**Actual behavior**

**Screenshots**

**Desktop (please complete the following information):**
 - OS: [e.g. Windows, Mac] Irrespective of Mac
 - Browser [e.g. Chrome, Safari] or Chrome or Safari it happens

**Smartphone (please complete the following information):**
 - Device: [e.g. iPhone6] Error
 - OS: [e.g. iOS8.1] Error
 - Browser [e.g. stock browser, Safari] Error

**Additional context**
I checked  https://docs.bigbluebutton.org/2.2/troubleshooting.html#freeswitch-fails-to-start-with-a-setscheduler-error  
**I DID IPV6 disabled  check** 

sudo ip addr | grep inet6
**Result:**
    inet6 ::1/128 scope host 
    inet6 2607:5300:203:4e7::/64 scope global 
    inet6 fe80::ec4:7aff:fe9a:de6c/64 scope link 
    inet6 fe80::42:c6ff:fe9a:1ffe/64 scope link 
    inet6 fe80::42:70ff:fed4:7b63/64 scope link 
    inet6 fe80::706f:beff:fec0:777b/64 scope link 
    inet6 fe80::d458:74ff:fe98:18f9/64 scope link

**Now I  DID FreeSWITCH fails to start with a SETSCHEDULER error CHECK:**
freeswitch.service - freeswitch
   Loaded: loaded (/lib/systemd/system/freeswitch.service; enabled; vendor prese
   Active: failed (Result: exit-code) since Sat 2021-05-01 23:28:57 UTC; 40min a
  Process: 32593 ExecStart=/opt/freeswitch/bin/freeswitch -u freeswitch -g daemo
 Main PID: 20497 (code=exited, status=0/SUCCESS)

May 01 23:28:57 ns539621 systemd[1]: freeswitch.service: Service hold-off time o
May 01 23:28:57 ns539621 systemd[1]: freeswitch.service: Scheduled restart job, 
May 01 23:28:57 ns539621 systemd[1]: Stopped freeswitch.
May 01 23:28:57 ns539621 systemd[1]: freeswitch.service: Start request repeated 
May 01 23:28:57 ns539621 systemd[1]: freeswitch.service: Failed with result 'exi
May 01 23:28:57 ns539621 systemd[1]: Failed to start freeswitch.

**Comment by erpsolns** [comment]
Linux ubuntu 4.15.0-135-generic #139-Ubuntu SMP Mon Jan 18 17:38:24 UTC 2021 x86_64 x86_64 x86_64 GNU/Linux


  Server	: 281828

  IPv4		: XXX.217.72.231

  IPv6		: 2607:XXXX:XX3:4e7::

  Hostname	: nsXXXXX21.ip-1X-2X7-72.net

IT SHOWS PROPER IPV4, IPV6 AND HOST NAME - BUT I CHANGED PARTS WITH XXX AS THIS IS PUBLIC FORUM

**Comment by erpsolns** [comment]
I tried everything stated in this forum and google forum. 1)I uninstalled the freeswitch module completely by apt-get purge freeSwitch and  bbb-install.sh. Still same error -2) I completely removed bigbluebutton by apt-get purge --autoremove --simulate bigbluebutton  and then bbb-install.sh - Still the same error. But got my server audio conference working back by this old and trusted method. Very simple method. Might have presumed by now.
Reinstalled the Ubuntu 18.04 and ran bbb-install.sh  - wow it is working great. Ha ha ha
